js/components/closed-caption/textEnhancementsTab.js

```diff
@@ -42,22 +44,36 @@ var TextEnhancementsTab = React.createClass({
     var textEnhancementItems = [];
     for(var i = 0; i < this.state.textEnhancements.length; i++) {
       //accent color
+      var isSelected = this.props.closedCaptionOptions.textEnhancement === this.state.textEnhancements[i];
       var selectedTextEnhancementStyle = {};
-      if (this.props.closedCaptionOptions.enabled && this.props.skinConfig.general.accentColor &&  this.props.closedCaptionOptions.textEnhancement == this.state.textEnhancements[i]) {
+      if (this.props.closedCaptionOptions.enabled && this.props.skinConfig.general.accentColor && isSelected) {
         selectedTextEnhancementStyle = {color: this.props.skinConfig.general.accentColor};
       }
+      var itemLabel = Utils.getLocalizedString(
+        this.props.language,
+        CONSTANTS.SKIN_TEXT[this.state.textEnhancements[i].toUpperCase()],
```

Comment:
Same as the font sizes. Right now there are textEnhancements: ["Uniform", "Depressed", "Raised", "Shadow"] defined in the initial state and the same values in the SKIN_TEXT constants. Can we combine these somehow?